xml/schema/Contact/Individual.xml

```diff
@@ -78,7 +78,7 @@
     <name>gender</name>
     <type>enum</type>
     <import>true</import>
-    <values>Female, Male, Transgender</values>
+    <values>Female, Male, Other</values>
```

Comment:
My understanding is that this change wasn't necessary as the 'gender' field was dropped, but I figured it should be consistent.